docs/reference/flow-yaml-schema-reference.md

```diff
@@ -0,0 +1,223 @@
+| `inputs.<input_name>`      | object    | The flow input definition. See [Flow input](#flow-input) for the set of configurable properties.                                                                                                                  |
+| `outputs`                  | object    | Dictionary of flow outputs. The key is a name for the output within the context of the flow and the value is the flow output definition.                                                                          |
+| `outputs.<output_name>`    | object    | The component output definition. See [Flow output](#flow-output) for the set of configurable properties.                                                                                                          |
+| `nodes`                    | array     | Sets of dictionary of individual nodes to run as steps within the flow. Node can use built-in tool or third-party tool. See [Nodes](#nodes) for more information.                                                 |
+| `node_variants`            | object    | Dictionary of nodes with variants. The key is the node name and value contains variants definition and `default_variant_id`. See [Node variants](#node-variants) for more information.                            |
+| `environment`              | object    | The environment to use for the flow. The key can be `image` or `python_requirements_txt` and the value can be either a image or a python requirements text file.                                                  |
+| `additional_includes`      | array     | Additional includes is a list of files that can be shared among flows. Users can specify additional files and folders used by flow, and Prompt flow will help copy them all to the snapshot during flow creation. |
+
+
+### Flow input
+
+| Key             | Type                                      | Description                                | Allowed values                                      |
+|-----------------|-------------------------------------------|--------------------------------------------|-----------------------------------------------------|
+| `type`          | string                                    | The type of flow input.                    | `int`, `double`, `bool`, `string`, `list`, `object` |
+| `description`   | string                                    | Description of the input.                  |                                                     |
+| `default`       | int, double, bool, string, list or object | The default value for the input.           |                                                     |
+| `is_chat_input` | boolean                                   | Whether the input is the chat flow input.  |                                                     |
```

Comment:
A new property has been added, is_chat_history
type: boolean
description: Whether the input is the chat history for chat flow.

Reply:
@lalala123123  maybe directly contribute to this PR. 
As only have one page under develop-a-flow/ feels lonely. 
You can add your page develop-chat-flow together with this.

Or @YingChen1996 can create that empty page first

Reply:
added